content/en/docs/concepts/workloads/pods/ephemeral-containers.md

```diff
@@ -80,18 +80,18 @@ you can view processes in other containers.
 
 ### Examples
 
-{{< caution >}}
+{{< note >}}
 The examples in this section require the `EphemeralContainers` [feature
-gate](/docs/reference/command-line-tools-reference/feature-gates.md) to be
-enabled and kubernetes client and server version of at least 1.16.
-{{< /caution >}}
+gate](/docs/concepts/workloads/pods/ephemeral-containers/) to be
```

Comment:
@verb, a few more links to fix.
Line 85, did you intend to link to the feature gates page, 
/docs/reference/command-line-tools-reference/feature-gates/